src/main/java/com/example/controller/MainMenuController.java

```diff
@@ -26,6 +39,20 @@ public class MainMenuController implements ChangeListener<ApplicationState> {
     private void bindToolbars(){
         graphToolsController.setDrawMenuController(drawMenuController);
         graphToolsController.setSimulationMenuController(simulationMenuController);
+        exportButton.setOnAction(e -> {
```

Comment:
Method bindToolbars is designed to pass controllers to their equivalents in toolbar. Setting buttons action should not be set here. Imo best option would be separate FileMenuController as I mentioned before.